src/product-service/handlers/getProductsById.test.ts

```diff
@@ -0,0 +1,54 @@
+import { getProductsById } from '../handlers/getProductsById';
+
+test('getProductsById with productId 1 returns frist obj from data.json with id 1 ', async () => {
```

Comment:
Missed describe

Reply:
More common to use the following pattern for tests

`it('should ....)`

Reply:
Done